tests/vmi_lifecycle_test.go

```diff
@@ -653,32 +662,52 @@ var _ = Describe("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:compon
 			AfterEach(func() {
 				cfgMap, err = virtClient.CoreV1().ConfigMaps(namespaceKubevirt).Get("kubevirt-config", options)
 				Expect(err).ToNot(HaveOccurred())
-				cfgMap.Data[virtconfig.FeatureGatesKey] = originalData
+				cfgMap.Data[virtconfig.FeatureGatesKey] = originalFeatureGates
 				_, err = virtClient.CoreV1().ConfigMaps(namespaceKubevirt).Update(cfgMap)
 				Expect(err).ToNot(HaveOccurred())
+
+				n, err := virtClient.CoreV1().Nodes().Get(node.Name, metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				n.SetLabels(originalLabels)
+				_, err = virtClient.CoreV1().Nodes().Update(n)
+				Expect(err).ToNot(HaveOccurred())
+
 				//FIXME improve the detection if virt-controller already received the config-map change
 				time.Sleep(time.Millisecond * 500)
 			})
 
-			It("[test_id:1639]the vmi with cpu.model matching a nfd label on a node should be scheduled", func() {
+			It("[test_id:1639]the vmi with cpu.model and cpu.features matching nfd labels on a node should be scheduled", func() {
```

Comment:
This is an overloaded test. We can't distinguish between the cpu model node selector causing the node to be scheduled or the Features.